### ISSUE TITLE
src-test doesn't work as it used to

### ISSUE THREAD

**Issue by janhancic** [BladeRunnerJS/brjs#1412]
This is with RC1

I have this folder structure:

```
apps/myapp/mybladeset/myblade/
  - test-acceptance/
    - src-test/
      - myTestUtils.js
    - tests/
      - MyTest.js
```

In `MyTest.js` I had this line:

```javascript
var myTestUtils = require('myapp/mybladeset/myblade/myTestUtils');
```

Which worked fine before RC1. After upgrading that failed with a BRJS exception saying it can't find the source module.

Moving the `src-test` folder into `myblade` directly "fixed" the problem.

Is this expected behaviour? If so, I couldn't find any documentation mentioning this change.

**Comment by andyberry88** [comment]
> Is this expected behaviour? If so, I couldn't find any documentation mentioning this change.

No, this is probably a regression following the API changes. Sounds like we need some more test coverage around this functionality.

**Comment by ioanalianabalas** [comment]
Fixed by #1423.
